.gitignore

```diff
@@ -60,3 +60,5 @@ nitunit.xml
 .metadata/*
 .github_data
 .DS_Store
+
+*/nit_compile/*
```

Comment:
Are you sure about the first wildcard? I think that it should be `**/nit_compile/` to match any subfolder depth.

Reply:
I'm sure of nothing, it patched my case which was indeed at a depth of 1. I'll try some variations and amend following this, thanks!

Reply:
You were right! I've updated this